src/Model/Behavior/SearchBehavior.php

```diff
@@ -30,6 +32,7 @@ class SearchBehavior extends Behavior
             'searchManager' => 'searchManager',
             'isSearch' => 'isSearch'
         ],
+        'emptyValues' => ['', false, null]
```

Comment:
This new option needs to be documented in docblock above and also in readme.

Reply:
Already documented